transform-plugins/widgets/ValueMapper-transform.json

```diff
@@ -5,18 +5,23 @@
   "configuration-groups": [
     {
       "properties": [
-        {
-          "widget-type": "textbox",
+	{
+          "name": "mapping",
           "label": "Mapping",
-          "name": "mapping"
+          "widget-type": "ds-multiplevalues",
+          "widget-attributes": {
+            "delimiter": ",",
```

Comment:
So this is not needed all the time but if you need some other character as delimiter (or values-delimiter) then it would be useful to specify it here. 
Otherwise it is not mandatory, the widget has default values for these attributes and they are the same as you had mentioned here.